starknet_py/net/full_node_client.py

```diff
@@ -193,9 +193,9 @@ async def estimate_fee(
                     "max_fee": convert_to_felt(tx.max_fee),
                     "version": hex(tx.version),
                     "signature": [convert_to_felt(i) for i in tx.signature],
-                    "nonce": convert_to_felt(
-                        0
-                    ),  # TODO: this will be used in the next cairo version
+                    "nonce": (
+                        convert_to_felt(tx.nonce) if tx.nonce is not None else None
```

Comment:
I'm not sure if this is 100% compliant with the RPC specs. I'd add a different TODO about verifying this.